src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__item {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: auto;
+    color: $color-main-accent;
+  }
+
+  &__code {
+    color: $color-secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-bottom: 16px;
```

Comment:
Better use one direction for margin, to avoid it's collapsing

Reply:
@IvanFesenko thank you for your advice.